apps/myaccount/src/components/shared/mobile-update-wizard.tsx

```diff
@@ -0,0 +1,421 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Field, Forms, useTrigger, Validation } from "@wso2is/forms";
+import { FormValidation } from "@wso2is/validation";
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Button, Divider, Message, Modal, Segment } from "semantic-ui-react";
+import { resendSMSOTPCode, updateProfileInfo, validateSMSOTPCode } from "../../api";
+import { EnterCode, QRCodeScan } from "../../configs";
+import { AlertInterface, AlertLevels } from "../../models";
+import { getProfileInformation } from "../../store/actions";
+import { useDispatch } from "react-redux";
+import { ProfileConstants } from "@wso2is/core/constants";
+
+/**
+ * Prop types for the SMS OTP component.
+ */
+interface MobileUpdateWizardProps {
+    onAlertFired: (alert: AlertInterface) => void;
+    closeWizard: () => void;
+    wizardOpen;
+    currentMobileNumber;
+    isMobileRequired;
+}
+
+/**
+ * Mobile number update section.
+ *
+ * @return {JSX.Element}
+ */
+export const MobileUpdateWizard: React.FunctionComponent<MobileUpdateWizardProps> =
+    (props: MobileUpdateWizardProps): JSX.Element => {
+    const { onAlertFired, closeWizard, wizardOpen, currentMobileNumber, isMobileRequired } = props;
+    const [step, setStep] = useState<number>(0);
+    const [verificationError, setVerificationError] = useState<boolean>(false);
+    const [resendSuccess, setResendSuccess] = useState<boolean>(false);
+    const { t } = useTranslation();
+    const [updateMobile, setUpdateMobile] = useTrigger();
+    const [submit, setSubmit] = useTrigger();
```

Comment:
```suggestion
    const { onAlertFired, closeWizard, wizardOpen, currentMobileNumber, isMobileRequired } = props;
    const [ step, setStep ] = useState<number>(0);
    const [ verificationError, setVerificationError ] = useState<boolean>(false);
    const [ resendSuccess, setResendSuccess ] = useState<boolean>(false);
    const { t } = useTranslation();
    const [ updateMobile, setUpdateMobile ] = useTrigger();
    const [ submit, setSubmit ] = useTrigger();
```

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1275/commits/375a1a7a8b040739aaae46cb9b9657e461a0bf70